From rpjday@crashcourse.ca  Fri Mar  7 08:29:46 2008
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Fri, 7 Mar 2008 11:02:00 -0500 (EST)
Subject: USB: g_printer.h does not need to be "unifdef"ed.
To: linux-usb@vger.kernel.org
Message-ID: <alpine.LFD.1.00.0803071100170.27973@localhost.localdomain>



Since the header file g_printer.h doesn't depend on __KERNEL__,
there's no need to unifdef it in the Kbuild file.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/Kbuild |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/usb/Kbuild
+++ b/include/linux/usb/Kbuild
@@ -3,5 +3,5 @@ header-y += cdc.h
 header-y += ch9.h
 header-y += gadgetfs.h
 header-y += midi.h
-unifdef-y += g_printer.h
+header-y += g_printer.h
 
